src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -41,7 +41,7 @@ public OpenCommandParser(Model notablyModel, CorrectionEngine<AbsolutePath> path
                 throw new ParseException("Path cannot be empty");
             }
         } else {
-            titles = argMultimap.getValue(PREFIX_TITLE).get();
+            titles = argMultimap.getValue(PREFIX_TITLE).get().trim();
```

Comment:
@ljiazh3ng I believe `argMultimap.getValue(PREFIX_TITLE).get()` should actually already return the trimmed version of the input right?